src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java

```diff
@@ -39,6 +39,10 @@
     // Yes/No field indicating if there are warnings to show
     public final String displayWarnings;
 
+    // Yes/No field indicating if the payment document control numbers are processed
+    @JsonProperty("awaitingDCNProcessing")
+    public final String awaitingDcnProcessing;
```

Comment:
In current form this PR can only be merged once all services have this field added to their exception record definitions in aat and prod.

Reply:
Yeah. As per the ticket, I'll have to merge this PR to demo branch first. After QA, I'll merge this pr to master